src/synthesize.jl

```diff
@@ -160,39 +159,74 @@ function synthesize(atm::ModelAtmosphere, linelist, λs::AbstractRange; metallic
 end
 
 """
-    get_absolute_abundances(metallicity, abundances, solar_abundances, solar_relative)
+    construct_abundances(metallicity, abundances)
 
-Calculate ``n_X/n_\\mathrm{total}`` for each element X given abundances in either [``X``/H] or 
-``A(X)`` form.  See [`synthesize`](@ref) for a detailed description of the arguments. Returns a 
-vector indexed by atomic number.
+Returns a 92 element vector containing abundances in ``A(X)`` (``\\log(X/H) + 12``) format for 
+elements from hydrogen to uranium.
+
+# Arguments
+You can provide either or both of:
+- `metallicity` (default: 0), i.e. [metals/H] is the ``\\log_{10}`` solar-relative abundance of elements heavier 
+   than He. It is overriden by `abundances`.  
```

Comment:
```suggestion
   than He. It is overriden by `abundances`, on a per-element basis.  
```

Not strictly necessary, but I think it helps add clarity.